packages/ibm-products/src/components/Card/Card.js

```diff
@@ -41,10 +42,14 @@ export let Card = forwardRef(
       // The component props, in alphabetical order (for consistency).
       actionIcons = defaults.actionIcons,
       actionsPlacement = defaults.actionsPlacement,
+      metadata = defaults.actionIcons,
```

Comment:
is there any specific reason why `metadata` is required and we can't just use `actionIcons`?

Reply:
when discussing with the PLG group, actionIcons was an icon you would take action on like in the other cards, but in this case it's more describing the card... so we thought metadata was a better prop name, but code-wise almost the same thing